doc/source/cluster/slurm.rst

```diff
@@ -21,51 +21,40 @@ Here are some community-contributed templates for using SLURM with Ray:
 
 .. _`Template script`: https://gist.github.com/pengzhenghao/b348db1075101a9b986c4cdfea13dcd6
 
+
 Starter SLURM script
 --------------------
 
 .. code-block:: bash
 
   #!/bin/bash
-
   #SBATCH --job-name=test
   #SBATCH --cpus-per-task=5
   #SBATCH --mem-per-cpu=1GB
-  #SBATCH --nodes=3
-  #SBATCH --tasks-per-node 1
-
-  worker_num=2 # Must be one less that the total number of nodes
+  #SBATCH --nodes=4
+  #SBATCH --tasks-per-node=1
+  #SBATCH --time=00:30:00
+  #SBATCH --reservation=test
```

Comment:
could you add a quick comment about what this does? this would be super helpful to other readers